### PR TITLE
Updated local-storage ci config to use go 1.16

### PR DESCRIPTION
**Pull request by priyanka19-98**
Signed off by: Priyanka Jiandani <pjiandan@redhat.com>

### CODE REVIEW

File: ci-operator/config/openshift/local-storage-operator/openshift-local-storage-operator-master.yaml [hunk 1]
```diff
@@ -31,15 +31,15 @@ base_images:
...
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: golang-1.16
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: golang-1.16
```
Comment (jsafrane) [inline ci-operator/config/openshift/local-storage-operator/openshift-local-storage-operator-master.yaml:42]: This is the golang that's used for unit tests. I am not sure about the other images above, they may be actually unused and everybody is afraid to remove them. IMO if you change just this `build_root:` golang to 1.16, it should be enough to get the 1.16 into unit tests and leave the rest as it is.

(someone should clean them up, but that can be in a following PR)

**Review by jsafrane — COMMENTED** [review]
Please update also `-4.8.yaml` and `-4.9.yaml` in the same directory

File: ci-operator/config/openshift/local-storage-operator/openshift-local-storage-operator-master.yaml (hunk 1 above)
Comment (priyanka19-98) [reply ci-operator/config/openshift/local-storage-operator/openshift-local-storage-operator-master.yaml:42]: done

**Pull request merged**